README.md

```diff
@@ -113,3 +113,12 @@ To fetch the articles from the RSS feed, run the following:
 ```sh
 docker exec -i zeeguu-api-core python /opt/Zeeguu-Core/tools/feed_retrieval.py
 ```
+
+Some RSS feed examples to get started:   
```

Comment:
Can you also make this a sub-header? (i.e.: ``#### RSS feed examples``).
And then make the below list to be a bullet list (add ``* `` at the start of each line).